drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -99,5 +102,16 @@ def enrollment_access_summaries(filters: nil)
           project_ids: filters&.project,
         )
     end
+
+    def staff_assignments
```

Comment:
do we need a permissions check here or is it assumed that if we can resolve the user, we have permission to resolve staff assignments too?

Reply:
Hm, that's a good question. If we were to add a permission check here, I think there's an easy way and a harder way:
- easy: You can only resolve this if you are the user making the request. This is more restrictive than necessary but would work, since currently this query is only used for populating "My Dashboard"
- harder: You can resolve all of the staff assignments for this user related to enrollments that you have permission to view. (We assume that if you are the current user then you have permission to view all of your assignees' enrollments)

I couldn't find anything about this in the [spec document](https://docs.google.com/document/d/17o9wMCrYR1Wo4vX5iLUqYMe1b7pZQjDdhFCQx6_T5-M/edit#heading=h.3trpwowucmf2) or the ticket, so I'll check in with Mary Ann

Reply:
Hm, on a little more investigation, the user [query](https://github.com/greenriver/hmis-warehouse/blob/9755f5b77568378ef4f160713db8579848c4d63d/drivers/hmis/app/graphql/types/hmis_schema/query_type.rb#L321) is already gated on what I think is the appropriate permission here. You can resolve a user if it's yourself, or if you have "can audit users" or "can impersonate users" permission.